.infra/application.properties

```diff
@@ -11,11 +11,8 @@ debezium.source.table.include.list=public.comment,public.comment_upvote,public.c
 debezium.source.column.exclude.list=public.post.tsv,public.post.placeholder,public.post.views,public.post.score,public.post.discussionScore,public.notification."readAt"
 debezium.source.plugin.name=pgoutput
 debezium.source.heartbeat.interval.ms=60000
-debezium.transforms=Reroute,filter
+debezium.transforms=Reroute
 debezium.transforms.Reroute.type=io.debezium.transforms.ByLogicalTableRouter
 debezium.transforms.Reroute.topic.regex=(.*)
 debezium.transforms.Reroute.topic.replacement=%topic%
-debezium.transforms.filter.type=io.debezium.transforms.Filter
```

Comment:
What is correct here?
I see in the previous PR you introduce it and here it's gone again?

Reply:
This removes it since we don't need filtering anymore after migration. It is planned as a one time thing currently to disable CDC and avoid many messages when we do the mass update of `flags` column.